interfaces/Portalicious/src/app/components/registration-page-layout/components/ignore-duplicates-dialog/ignore-duplication-dialog.component.html

```diff
@@ -0,0 +1,44 @@
+<app-confirmation-dialog
+  #ignoreDuplicationDialog
+  header="Ignore duplication"
+  i18n-header
```

Comment:
```suggestion
  i18n-header="@@ignore-duplication"
```

This will prevent 1 duplicate entry in the messages.xlf file.